tap-agent/src/agent/sender_account.rs

```diff
@@ -87,7 +87,13 @@ type Balance = U256;
 pub enum ReceiptFees {
     NewReceipt(u128, u64),
     UpdateValue(UnaggregatedReceipts),
-    RavRequestResponse(anyhow::Result<(UnaggregatedReceipts, Option<SignedRAV>)>),
+    RavRequestResponse(
+        (
+            UnaggregatedReceipts,
+            Option<SignedRAV>,
+            Result<(), anyhow::Error>,
+        ),
```

Comment:
This tuple could be: `(UnaggregatedReceipts, anyhow::Result<Option<SignedRAV>>)`